Kerberos.NET/BinaryExtensions.cs

```diff
@@ -4,19 +4,33 @@
 
 namespace Kerberos.NET
 {
-    public static class BinaryExtensions
+    internal static class BinaryExtensions
     {
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        internal static byte[] TryGetArrayFast(ReadOnlyMemory<byte> bytes)
+        public static byte[] TryGetArrayFast(this ReadOnlyMemory<byte> bytes)
```

Comment:
This method is huge, so it should not inline. I'm pretty sure that perf is better without inlining as the code bloat (due inlining) is gone.

And I changed this to an extension methods -- it's in a class with `Extensions`-suffix :wink: